api/window/hotkey.js

```diff
@@ -162,6 +167,12 @@ export class Bindings extends EventTarget {
     const { id } = event.data ?? {}
     const binding = this.get(id)
 
+    if (event.timeStamp < this.#clock) {
```

Comment:
is timeStamp always defined?

Reply:
yep it is on the [`Event`](https://developer.mozilla.org/en-US/docs/Web/API/Event/timeStamp) prototype